musings/intel-tdx-attestation.md

```diff
@@ -0,0 +1,231 @@
+| Certification Data | Variable        | Byte Array  | PCK Leaf Cert/ PCK Cert Chain                                                                                        |
+
+As of now, only PCK Cert Chain is supported by Intel SGX DCAP Library
+
+### TDX Trust Anchors and Reference Values Management
+
+To assist the end-users to build attestation service, Intel has released a set of
+[Intel-SGX Data Centre Primitives](https://github.com/intel/SGXDataCenterAttestationPrimitives)
+for Linux and 
+[Certificate Retrieval Service](https://api.portal.trustedservices.intel.com/provisioning-certification)
+that enables platform owners and users to retrieve certificates for
+an Intel SGX enabled platform. 
+
+**Warning**: We think, the below set of API will need some modification to adjust to the new TDX Quote
+Structure (V4) as detailed above. The documentation that follows in this section outlines the general
+principle which will be followed to obtain the Trust Anchors and Reference Values for Intel-TDX enabled
+platforms.
+
+- Register Platform:
+This API allows to register a multi-package SGX platform. If the registration is successful, then platform
+provisioning root keys are stored in the back-end.
+
+- Download the Provisioning Certificate Root CA Certificate: 
+Intel Root Certificate can be downloaded from the Intel provided web-site.
+
+- Download PCK Certificates for a given Platform Manifest: 
+The Get PCK Certificates API allows requesting PCK Certificates for all configured TCB Levels 
+for a given platform using the given Platform Identifier or a Platform Manifest
+
+- Get TDX TCB information: 
+Retrieve TDX TCB information for a given FMSPC. The identifier FMSPC can be obtained from the 
+retrieved PCK Certificate
+
+The above information for a given Intel platform can be provisioned inside Veraison Store. The information can be cached and retrieved at the time of Verification to execute the Verification Algorithm given below.
+
+
+### TDX Quote Verification - Platform 
+
+#### Validating Evidence Integrity
+
+* Verify the signature on the Quote using the AK supplied in the Quote Signature Structure
+
+* Extract the PCK Certificate Chain from the supplied Quote
+
+* Using the AK and PCK Cert Chain from the Quote, Verify that the AK and the Cert Chain is rooted fully to
+the root certificate supplied from Intel
+
+* Using PCK Certificate obtain the FMSPC Value for the given platform
+
+2. Comparing Reference Values
+
+* From the Endorsement Store obtain TCB Information for the Platform. Specifically tdxtcbcomponents array.
+
+* Compare all the SGX TCB Comp SVNs retrieved from PCK Certificate(1 to 16) with the corresponding values 
+of SVNs in tdxtcbcomponents array. If the Certificate SVN Values are greater that the ones stored in the Endorsement Store, the proceed below. Otherwise move to the next TCB Level. If none of them are greater then it indicates
```

Comment:
Is "greater than" correct here, or is it meant to be "greater than or equal to"?